### ISSUE TITLE
Implement CloudWatch Scheduled Events as an event source in @aws-cdk/aws-lambda-event-sources

### ISSUE THREAD

**Issue by nathanpeck** [aws/aws-cdk#1648]
CloudWatch scheduled events are super powerful for creating Lambda workloads that execute on a cron schedule. It's a pretty common pattern for kicking off workloads at a specific time, or regularly polling, regenerating a report, etc:

https://docs.aws.amazon.com/lambda/latest/dg/with-scheduled-events.html

**Comment by eladb** [comment]
Copy: @sam-goodwin 

You can also set it up through the event rule (this is likely what the EventSource will do under the hood):

```ts
import events = require('@aws-cdk/aws-events');

new events.EventRule(this, 'Schedule', {
  scheduleExpression: '...',
  targets: [ lambda ]
});
```

**Comment by nathanpeck** [comment]
Ah nice, I'll use that approach for now then!

**Comment by rix0rrr** [comment]
I have an open PR for this. Let me dig it up

**Comment by rix0rrr** [comment]
I would say this is the "right" way? The event sources model seems a bit odd to me, tbh

**Comment by eladb** [comment]
The event sources model is "lambda-centric", which we know some of our customers are. I don't see it as a problem to have more than one way to achieve something on our system, based on where people are coming from.

**Comment by moravcik** [comment]
I was using the approach as @eladb suggested, but with recent updates in `@aws-cdk/aws-events` I am getting type error: 
    
    Argument of type 'Function' is not assignable to parameter of type 'IRuleTarget'. Property 'bind' is missing in type 'Function' but required in type 'IRuleTarget'.

Is there any workaround or alternative way how to achieve the same as before? Thank you in advance

**Comment by eladb** [comment]
@moravcik rule targets are now implemented by "integration classes". In your case, use `@aws-cdk/aws-events-targets.LambdaTarget`

**Comment by moravcik** [comment]
@eladb thank you, this is exactly what I needed.

**Comment by jderose9** [comment]
This appears to have been resolved in the 1.10.0 release.
